resource/StandardResources.md

```diff
@@ -0,0 +1,40 @@
+# Standard Resources
+
+This page lists the standard resource types in OpenCensus.
+
+## Kubernetes (k8s)
+
+Resources specific to Kubernetes:
+
+### `k8s.io/container`
+**Description:** A Kubernetes container instance.
```

Comment:
Do we need container fields like these: https://github.com/elastic/ecs#-container-fields

Reply:
We can add more later, this is a minimum set that we picked.

Reply:
in my head there are two types of attributes. Phisical model attrubutes which describe things the way k8 understand it. And semantical level attributes - generalized settings. I'm ok to adding mode attributes later for phisical model. 

Separate question - do we want semantic level data attributes defined? 

To be more specific - it will be great to define generic fields like:

`container.name` = `k8s.io/container/name`

and looking at https://github.com/elastic/ecs#-host-fields

`host.id` = `cloud.google.com/gce/instance_id` = `aws.com/ec2/instance_id`

Reply:
Tried to generalize more these names.